packages/playground/website/public/preview-referrer-pr.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+$referrer = $_SERVER['HTTP_REFERER'];
+$prNumber = null;
+
+if (preg_match('/github\.com\/WordPress\/gutenberg\/pull\/(?<prNumber>\d+)/i', $referrer, $matches)) {
+    $prNumber = $matches['prNumber'];
+    header('Location: ./gutenberg.html?pr=' . $prNumber);
```

Comment:
with #1761 would it be helpful to cut out the redirect and launch `/?core-pr=${prNumber}`?

Reply:
That was my first thought, too, but `/gutenberg.html` will tell you when the artifact isn't ready yet while a direct link would just not load the PR.

Reply:
a possible enhancement for #1761 down the line 😉